lib/transport/transport.ts

```diff
@@ -33,6 +42,13 @@ export function marshallTransform(jsValue: any, context: transportable.Transport
         if (constructorName !== 'Object') {
             if (typeof jsValue['cid'] === 'function') {
                 return <transportable.Transportable>(jsValue).marshall(context);
+            } else if (_verifiedBuiltInTypeWhitelist.has(constructorName)) {
```

Comment:
no _verified prefix? since whiteList already have this semantics.

Reply:
yes, just realize that

---
In reply to: [155905912](https://github.com/Microsoft/napajs/pull/137#discussion_r155905912) [](ancestors = 155905912)